src/otx/tools/converter.py

```diff
@@ -178,6 +178,10 @@
         "task": OTXTaskType.ANOMALY_CLASSIFICATION,
         "model_name": "stfpm",
     },
+    "ote_anomaly_classification_uflow": {
```

Comment:
I think we need to add ANOMALY task here, not ANOMALY_CLASSIFICATION. Again, I hope we will remove other task types in the future.